tests/foreman/ui/test_activationkey.py

```diff
@@ -211,378 +158,236 @@ def test_positive_create_activation_key_3(self, env_name):
             # add Host Collection
-            self.activationkey.add_host_collection(name, host_col['name'])
-            result = self.activationkey.find_element(
-                common_locators['alert.success'])
-            self.assertIsNotNone(result)
+            self.activationkey.add_host_collection(name, host_col.name)
+            self.assertIsNotNone(self.activationkey.find_element(
+                common_locators['alert.success']))
 
             # check added host collection is listed
-            self.activationkey.wait_until_element(
-                tab_locators['ak.host_collections.list']
-            ).click()
+            self.activationkey.click(tab_locators['ak.host_collections.list'])
             strategy, value = tab_locators['ak.host_collections.add.select']
             host_collection = self.activationkey.wait_until_element(
-                (strategy, value % host_col['name']))
+                (strategy, value % host_col.name))
             self.assertIsNotNone(host_collection)
 
-    def test_positive_create_activation_key_6(self):
-        """@Test: Create Activation key with default Usage limit (Unlimited)
-
-        @Feature: Activation key - Positive Create
-
-        @Steps:
-        1. Create Activation key with default Usage Limit (Unlimited)
-        using valid Name, Description, Environment and Content View
-
-        @Assert: Activation key is created
-
-        @BZ: 1078676
-
-        """
-        name = gen_string('alpha', 10)
-        with Session(self.browser) as session:
-            make_activationkey(
-                session,
-                org=self.org_name,
-                name=name,
-                env=ENVIRONMENT,
-                description=gen_string('alpha', 16)
-            )
-            self.assertIsNotNone(self.activationkey.search_key(name))
-
-    def test_positive_create_activation_key_7(self):
+    def test_positive_create_activation_key_with_usage_limit(self):
         """@Test: Create Activation key with finite Usage limit
 
         @Feature: Activation key - Positive Create
 
-        @Steps:
-        1. Create Activation key with finite Usage Limit (Not Unlimited)
-        using valid Name, Description, Environment and Content View
-
         @Assert: Activation key is created
 
-        @BZ: 1078676
-
         """
-        name = gen_string('alpha', 10)
-        description = gen_string('alpha', 10)
-        limit = '6'
+        name = gen_string('alpha')
         with Session(self.browser) as session:
             make_activationkey(
                 session,
-                org=self.org_name,
+                org=self.organization.name,
                 name=name,
                 env=ENVIRONMENT,
-                limit=limit,
-                description=description
-            )
-            self.assertIsNotNone(self.activationkey.search_key(name))
-
-    def test_positive_create_activation_key_8(self):
```

Comment:
That is a default scenario which is definitely covered in scope of basic scenario `test_positive_create_activation_key_with_different_names`